test/javascripts/acceptance/question-answer-test.js

```diff
@@ -190,8 +202,38 @@ acceptance("Discourse Question Answer - logged in user", function (needs) {
     );
   });
 
+  test("editing a comment", async function (assert) {
```

Comment:
Do you test failures in acceptance tests?

Reply:
To be honest usually I don't because our JS acceptance test makes it hard to return a response code that is not 200.